codex-rs/app-server/tests/suite/v2/account.rs

```diff
@@ -29,6 +34,7 @@ fn create_config_toml(
     codex_home: &Path,
     forced_method: Option<&str>,
     forced_workspace_id: Option<&str>,
+    requires_openai_auth: Option<bool>,
```

Comment:
Maybe we don't feel this is necessary yet, but consider:


```suggestion
#[derive(Default)]
struct CreateConfigTomlParams {
forced_method: Option<&str>,
forced_workspace_id: Option<&str>,
    requires_openai_auth: Option<bool>,
}

fn create_config_toml(
codex_home: &Path,
params: CreateConfigTomlParams,
)
```

then you could:

```
let params = CreateConfigTomlParams { forced_method: Some(""), ..Default::default()) }
```

maybe not worth it, but fyi?

Reply:
sg - this seems neater. this is good learning for me!